src/components/Popover/_theme.scss

```diff
@@ -11,6 +11,7 @@ $color: var(--rui-Popover__color);
 $background-color: var(--rui-Popover__background-color);
 $box-shadow: var(--rui-Popover__box-shadow);
 
-$arrow-width: 1rem;
+// 1 rem (16px) + 2px (for insetting the arrow)
+$arrow-width: 18px;
```

Comment:
You can actually use the calculation (and keep the rem units) and delete the comment:

```suggestion
$arrow-width: calc(1rem + #{$arrow-safe-overlap + $arrow-gap});
```

Plus:

```scss
$arrow-safe-overlap: 1px;
$arrow-gap: 1px;
```

Reply:
BTW, the one extra pixel creates this gap between the arrow and the parent element which is something I also wanted to add. Thank you! 👍🏻

<img width="646" alt="image" src="https://github.com/user-attachments/assets/e05e86a5-5f2a-4cb5-9273-5078657ec260">